learn/keeping-ballerina-up-to-date.md

```diff
@@ -230,5 +230,5 @@ Downloading ballerina-tool-0.8.1 100% [====================================] 1/1
 
 Updated to latest tool version: 0.8.1
 Cleaning old files...
-Ballerina tool updated successfully
+Ballerina Tool updated successfully
```

Comment:
This is probably the output from the `ballerina update` command. So we probably have to fix it there (first?). @keizer619

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/27407 to fix in the Tool outputs.